buildSrc/src/main/kotlin/com/datadog/gradle/plugin/gitdiffs/GitDiffsProvider.kt

```diff
@@ -0,0 +1,27 @@
+package com.datadog.gradle.plugin.gitdiffs
+
+import com.datadog.gradle.utils.execShell
+import org.gradle.api.Project
+
+open class GitDiffsProvider(val project: Project) {
+
+    open val diffs: List<String> by lazy {
+        val currentBranch = currentBranch()
+        val targetBranch = targetBranch()
+        println("Current branch $currentBranch")
+        project.execShell("git", "diff", "--name-only", currentBranch, targetBranch)
+    }
+
+    private fun targetBranch(): String {
+        val targetBranch = System.getProperty("ghprbTargetBranch")
```

Comment:
Is this property something specific to the plugin you inspired yourself from ?  Maybe there's an evn variable that's accessible on gitlab CI's containers that would make more sense using ?

Reply:
That's what I wanted to ask you today. If you know a way we can get this from Gitlab.